index.html

```diff
@@ -0,0 +1,62 @@
+<html lang="en" data-lt-installed="true">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome Books</title>
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <nav class="nav-bar">
+    <h3>Awesome books</h3>
+    <ul class="menu">
+      <li><a href="#" class="menua">List</a></li>
+      <li><a href="#" class="menub">Add New</a></li>
+      <li><a href="#" class="menuc">Contact</a></li>
+    </ul>
+  </nav>
+  <div class="date_container">
+    <div class="date">
+    </div>
+  </div>
+  <main class="list">
+    <h1>All awesome books</h1>
+    <div id="list" class="list">
+      <ul class="book-list" id="bookList"></ul>
+    </div>
+    </div>
+    <div class="copyright-div">
+      <p class="copyright">Copyright .. </p>
+    </div>
```

Comment:
- [ OPTIONAL] For semantic reasons, I recommend putting this below and wrapping it with the `footer` tag rather than a `div`. And to fix it at the bottom of the page, I recommend updating the style as shown below. :point_down: 

```
.copyright  {
    border: 3px solid black;
    padding: 10px;
    position: fixed;
    bottom: 0px;
    left: 0px;
    right: 0px;
    margin: 5px 20px;
}

```
#### Your result will look like this :point_down: 
![Screenshot from 2023-05-11 16-53-13](https://github.com/CarlosZ96/Awesome-Books-ES6/assets/89228743/b4e4acd0-01bc-47f5-af03-533791a1a295)


#### Currently this :point_down:  is what we have.

![Screenshot from 2023-05-11 16-54-32](https://github.com/CarlosZ96/Awesome-Books-ES6/assets/89228743/9942ed42-1b97-4922-88bb-53dec670ae6e)